src/theme/DocItem/RightSidebar/TOC/index.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
```

Comment:
I good amount of this comes from the swizzled DocItem https://github.com/dfuzr/libra-website/pull/41/files#diff-a8a8091466f5e231f114b0f879ee3e26R24 (line 24-54)